docs/source/license_link.md

```diff
@@ -0,0 +1,4 @@
+# LICENSE
```

Comment:
This shouldn't be here, but in LICENSE

Reply:
well, then I don't have to change the license file itself and keep it as it is...

Reply:
Uhm...I'm not sure what you mean? You mean only adding a # heading to LICENSE.md? Yeah I think that might be enough, 😂 just thought that LICENSE.md looked weird, cause the standard is just LICENSE but I guess it's fine

Reply:
ahh, now I know what you mean! Yes, true, LICENCE.md doesn't really make sense, it rather a plain txt file, no markdown! I'll change it soonish! :) Thx